che-editors.yaml

```diff
@@ -208,25 +208,25 @@ editors:
               targetPort: 3100
               exposure: public
               secure: true
-              protocol: http
+              protocol: https
             - name: code-redirect-1
               targetPort: 13131
               exposure: public
-              protocol: http
+              protocol: https
               attributes:
                 discoverable: false
                 urlRewriteSupported: false
```

Comment:
I haven't had time to verify this directly and so I'm not sure it's an _actual_ problem, but does this cause issues when running on Kubernetes? Since these endpoints have `urlRewriteSupported: false`, I believe Che will support them via plain ingresses, and I'm not sure if Che is able to configure TLS there.

Reply:
yeah, good, I need to check this on minikube

Reply:
tried to verify on minikube but it looks like Che is not supported on this arch in general:

```
 ❯ Eclipse Che Operator pod bootstrap
      ✔ Scheduling...[OK]
      ✖ Downloading images
        → Failed to download image, reason: ErrImagePull, message: rpc error: co
…
        Starting
      Create ValidatingWebhookConfiguration org.eclipse.che
      Create MutatingWebhookConfiguration org.eclipse.che
      Create CheCluster Custom Resource
Error: Command server:deploy failed with the error: Failed to download image, reason: ErrImagePull, message: rpc error: code = Unknown desc = no matching manifest for linux/arm64/v8 in the manifest list entries. See details: /Users/ibuziuk/Library/Caches/chectl/error.log. Eclipse Che logs: /var/folders/87/t16qbhjd6sn7cz51_g716sh40000gn/T/chectl-logs/1697121281363.
    at newError (/Users/ibuziuk/chectl/lib/utils/utls.js:39:19)
    at wrapCommandError (/Users/ibuziuk/chectl/lib/utils/command-utils.js:53:32)
    at Deploy.<anonymous> (/Users/ibuziuk/chectl/lib/commands/server/deploy.js:82:65)
    at Generator.throw (<anonymous>)
    at rejected (/Users/ibuziuk/chectl/node_modules/tslib/tslib.js:165:69)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
Cause: Error: Failed to download image, reason: ErrImagePull, message: rpc error: code = Unknown desc = no matching manifest for linux/arm64/v8 in the manifest list entries.
    at /Users/ibuziuk/chectl/lib/tasks/pod-tasks.js:149:39
    at Generator.next (<anonymous>)
    at fulfilled (/Users/ibuziuk/chectl/node_modules/tslib/tslib.js:164:62)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

However, Smoke Test PR check should cover the minikube verification. Adding @dmytro-ndp for PR approval

Reply:
I had gotten Network Error when tried to start workspace of Eclipse Che Next on minikube which is using external plugin registry with URL https://pr-check-1810-che-plugin-registry.surge.sh/v3 given by @ibuziuk :
![Screenshot from 2023-10-17 13-30-08](https://github.com/eclipse-che/che-plugin-registry/assets/1197777/13cf6ec7-8130-43f5-8c64-8f639882ada0)

Reply:
BTW: the error `Failed to fetch a devfile from URL: https://pr-check-1810-che-plugin-registry.surge.sh/v3/plugins/che-incubator/che-code/latest/devfile.yaml, reason: Error: Network Error` has reproduced on OpenShift 4.10 as well:
![Screenshot from 2023-10-17 16-15-29](https://github.com/eclipse-che/che-plugin-registry/assets/1197777/28a14e72-cab1-4c63-88ed-690397d387a8)

Reply:
@olexii4 @akurinnoy folks, it looks like in general it is not possible to reference external plugin registry from the CR since we would hit CORS on dashboard end, right?

Reply:
It looks like CORS requests are disabled on PR checks like https://pr-check-1810-che-plugin-registry.surge.sh/v3 
@svor do you happen to know if it is expected?

Reply:
I've just tested it on minikube (the workspace has been started, all plugins have been installed):
![screenshot-192 168 49 2 nip io-2023 10 30-17_03_01](https://github.com/eclipse-che/che-plugin-registry/assets/1271546/a0ce7047-c542-4549-9ca1-3f7a52525b08)
![screenshot-192 168 49 2 nip io-2023 10 30-17_02_25](https://github.com/eclipse-che/che-plugin-registry/assets/1271546/98217822-e89a-4be2-91f9-4fd1d384c97c)